buildbot-buildmaster/buildbot/secrets/README.md

```diff
@@ -0,0 +1,7 @@
+This is the place for
+- htpasswd file
+- github access token
+
+It seems that you have to provide the passwords in htpasswd as clear text for buildbot, e.g. you@email.provider:password.
```

Comment:
It seems? Please clarify. If they have to be specified this way, "it seems" should be removed

Reply:
Removed.